### DISCUSSION TITLE
Rich text excerpts

### DISCUSSION THREAD

**Discussion by alanfluff** [GetPublii/Publii#1396] [Help] [answered]
Hi, Is there some place in the docs where it explains how (when modifying the blank theme) to enable rich text excerpts -- so when on the homepage you see a set of articles, each with a title and a short extract, any HTML in the extract part is respected, not stripped? Thanks Publii ppl : )

**Comment by bobmitro** [comment, marked as answer]
To display the content of an article with HTML in an excerpt, you must use the "Read More" button in the editor. This will ensure that everything up to its placement is included in the excerpt.

Also, if you want to display the whole post body instead of the excerpt, you should edit the theme source file, for example, the index.hbs, and replace the `{{{excerpt}}}` with `{{{text}}}`

**Reply by alanfluff** [reply]
Brilliant. Thank you Bob.

Deeply impressed with Publii already and I've only edited about 5 lines in a copy of the blank theme so far. My first departure from the world of LAMP in many years and Publii has the same feeling of deep quality Processwire gave me when I stumbled on it. 

Thank you and other contributors for this great system.
